solidity/core/contracts/XAppConnectionManager.sol

```diff
@@ -13,16 +14,16 @@ import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
  * @notice Manages a registry of local Inbox contracts for remote Outbox
  * domains.
  */
-contract XAppConnectionManager is Ownable {
+contract XAppConnectionManager is IXAppConnectionManager, Ownable {
     // ============ Public Storage ============
 
     // Outbox contract
-    IOutbox public outbox;
```

Comment:
I like this pattern but the `override` seems so awkward on a variable
fwiw in solidity 8 this is no longer required for interfaces
https://github.com/ethereum/solidity/pull/11628

Reply:
Yeah it does. Unfortunately we need `outbox` in the interface so that `XAppConnectionClient` can use it.